evaluate_tflite.py

```diff
@@ -36,7 +36,8 @@
 
 def tflite_worker(model, scorer, queue_in, queue_out, gpu_mask):
     os.environ['CUDA_VISIBLE_DEVICES'] = str(gpu_mask)
-    ds = Model(model, BEAM_WIDTH)
+    ds = Model(model)
+    ds.setBeamWidth(BEAM_WIDTH)
```

Comment:
I'm wondering if we should keep those ? They were here because that was the only solution, but I would think we don't need them and/or re-use existing CLI flags?

Reply:
I'm ambivalent, I can remove them if you want.

Reply:
We already source `FLAGS` right, so it would make sense to re-use them there and avoid hard-coding magic constants that would override what ends up in the exported model that we want to be default.

Reply:
@reuben Re-thinking again, and I don't want to be painful but I still get convinced we should remove them: this code is supposed to evaluate, adding flags is just going to add mess. I really think we should just remove `BEAM_WIDTH` as well as `LM_ALPHA` and `LM_BETA`.